conversion_sr14_sr15/conversion_sr14_sr15.py

```diff
@@ -0,0 +1,143 @@
+import pandas as pd
+import yaml
+import os
+
+class ConfigLoader:
+    def __init__(self, config_path):
+        self.config_path = config_path
+        self.config = self.load_config()
+
+    def load_config(self):
+        os.chdir(os.path.dirname(os.path.abspath(__file__)))
+        with open(self.config_path, 'r') as yamlfile:
+            return yaml.load(yamlfile, Loader=yaml.FullLoader)
+
+class DataLoader:
+    def __init__(self, config):
+        self.config = config
+        self.hh_s14 = None
+        self.per_s14 = None
+        self.mgra_xwalk = None
+        self.landuse_s14 = None
+        self.landuse_abm3 = None
+        self.load_data()
+
+    def load_data(self):
+        os.chdir(self.config['input']['input_dir'])
+        self.hh_s14 = pd.read_csv(self.config['input']['filenames']['households'])
+        self.per_s14 = pd.read_csv(self.config['input']['filenames']['persons'])
+        self.mgra_xwalk = pd.read_csv(self.config['input']['filenames']['mgra_xwalk'])
+        self.landuse_s14 = pd.read_csv(self.config['input']['filenames']['land_use'])
+        self.landuse_abm3 = pd.read_csv(self.config['input']['filenames']['land_use_abm3'])
+
+class Converter:
+    def __init__(self, data_loader):
+        self.data_loader = data_loader
+        self.mgra_xwalk_dict = self.data_loader.mgra_xwalk.set_index('MGRA13')['MGRA15'].to_dict()
+
+    def convert_hh(self):
+        hh_s14 = self.data_loader.hh_s14
+        hh_s14['mgra'] = hh_s14['mgra'].replace(self.mgra_xwalk_dict)
```

Comment:
How about column "taz"? Should it also be converted from SR14 to SR15 using a crosswalk file?